### DISCUSSION TITLE
How to modify OpenMP enabled; Number of OpenMP Threads is 1

### DISCUSSION THREAD

**Discussion by goodman249** [firemodels/fds#12716] [General]
Fire Dynamics Simulator

 Current Date     : March 20, 2024  20:01:38
 Revision         : FDS6.7.4-0-gbfaa110-release
 Revision Date    : Fri Mar 6 10:17:46 2020 -0500
 Compiler         : Intel ifort 19.0.4.243
 Compilation Date : Mar 09, 2020 16:12:17

 MPI Enabled;    Number of MPI Processes:      24
 OpenMP Enabled; Number of OpenMP Threads:      4

 MPI version: 3.1
 MPI library version: Intel(R) MPI Library 2019 Update 4 for Linux* OS

How to modify OpenMP enabled; Number of OpenMP Threads is 1，thank you

**Comment by LordMicidial** [comment]
You'll probably have to set the grid according to reach a good scalability (that depends also on your PC infrastructure available). I think i've learnt the most on such a topic here: https://www.youtube.com/watch?v=sMQwgKK_GYM&ab_channel=CFDFEASERVICE this YouTube channel is very effective!

**Comment by drjfloyd** [comment]
FDS User's Guide Chapter 3
